workspaces/README.md

```diff
@@ -1,4 +1,14 @@
-## PrairieLearn Workspace Images
+## PrairieLearn workspace images
 
-Within this directory are a set of docker containers that are intended to be run inside of workspaces.
-These docker containers may be modified as needed on a per-course basis.
+Within this directory are a set of Docker images that are intended to be run inside of workspaces.
+These Docker images may be modified as needed on a per-course basis.
```

Comment:
Final cleanup nits (I know this is unrelated to your changes, but as long as we're improving things...):
- These images aren't run _inside of_ workspaces, but rather are themselves the workspace images.
- Given that this file doesn't (and IMO shouldn't) contain comprehensive information about _how_ to actually modify the images, I'm inclined to say we should just drop the second sentence entirely.

```suggestion
This directory contains source files for Docker images that are intended to be used as [PrairieLearn workspaces](https://prairielearn.readthedocs.io/en/latest/workspaces/).
```